velox/type/Timestamp.cpp

```diff
@@ -159,6 +159,15 @@ const date::time_zone& Timestamp::defaultTimezone() {
   return *kDefault;
 }
 
+const int64_t Timestamp::getTimezoneOffset(const date::time_zone* timezone) {
+  auto now = std::chrono::system_clock::now();
```

Comment:
This is not correct, should use the time in data not the system time now

Reply:
It has been refactored to take from `Timestamp`.